### PR TITLE
Removed system run id from tables, views and stored procs

### PR DESCRIPTION
**Pull request by andrewsparkes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ Scripts to create/update the schema required for the cherrytrack database.
   * [Configurations](#configurations)
   * [Select unpicked wells for a source barcode e.g. if the source is a partial](#selects-unpicked-wells-for-a-source-barcode-eg-if-the-source-is-a-partial)
   * [Select empty well coordinates in a destination plate barcode](#select-empty-well-coordinates-in-a-destination-plate-barcode)
-  * [Example usage queries for use case 1 from the URS](#example-usage-queries-for-use-case-1-from-the-URS)
```
Comment (harrietc52) [inline README.md:19]: Does this want to be removed?

File: stored_procedures/create_run_record.sql [hunk 2]
```diff
@@ -53,7 +51,6 @@ BEGIN
       SELECT id FROM `automation_systems`
       WHERE automation_system_name = input_automation_system_name
     ),
-    input_system_run_id,
```
Comment (harrietc52) [inline stored_procedures/create_run_record.sql:56]: Is both `start_time` and `created_at` needed? Does `start_time` get updated later? And should `updated_at` be set to now() on creation, or should it be left empty?

File: stored_procedures/create_run_record.sql [hunk 3]
```diff
@@ -62,7 +59,7 @@ BEGIN
     now()
   );
 
-  SELECT LAST_INSERT_ID() INTO @automationSystemRunId;
+  SELECT LAST_INSERT_ID() INTO output_automation_system_run_id;
```
Comment (harrietc52) [inline stored_procedures/create_run_record.sql:62]: Should we keep the `@` to show its a user-defined variable? Local variables don't have the @ prefix

**Review by harrietc52 — APPROVED** [review]
Looks good! A few minor questions. Updated the schema diagram too

File: README.md (hunk 1 above)
Comment (andrewsparkes) [reply README.md:19]: yep, gone

File: stored_procedures/create_run_record.sql (hunk 3 above)
Comment (andrewsparkes) [reply stored_procedures/create_run_record.sql:62]: The examples I saw have the @ for internal (local) variables but not for the IN / OUT parameters. So think that's correct but not certain.

File: stored_procedures/create_run_record.sql (hunk 2 above)
Comment (andrewsparkes) [reply stored_procedures/create_run_record.sql:56]: Not sure. One is the PC time the run started and the other database insert time. If the first thing they do on starting the run is insert the row maybe it's fine without.
What's the standard with updated_at? does it get set same as created_at on first insert normally?

**Pull request merged**